charts/function-mesh-operator/README.md

```diff
@@ -20,34 +20,35 @@ The Function Mesh operator Helm chart for Kubernetes
 
 | Repository | Name | Version |
 |------------|------|---------|
-| (Built-in) | admission-webhook | 0.2.7 |
+|  | admission-webhook | 0.2.8 |
 
 ## Values
 
-| Key | Type | Default |
-|-----|------|---------|
-| admissionWebhook.enabled | bool | `true` |
-| controllerManager.affinity | object | `{}` |
-| controllerManager.autoFailover | bool | `true` |
-| controllerManager.configFile | string | `"/etc/config/config.yaml"` |
-| controllerManager.create | bool | `true` |
-| controllerManager.enableLeaderElection | bool | `true` |
-| controllerManager.healthProbe.port | int | `8000` |
-| controllerManager.metrics.port | int | `8080` |
-| controllerManager.nodeSelector | object | `{}` |
-| controllerManager.pprof.enable | bool | `false` |
-| controllerManager.pprof.port | int | `8090` |
-| controllerManager.replicas | int | `1` |
-| controllerManager.resources.requests.cpu | string | `"80m"` |
-| controllerManager.resources.requests.memory | string | `"50Mi"` |
-| controllerManager.selector | list | `[]` |
-| controllerManager.serviceAccount | string | `"function-mesh-controller-manager"` |
-| controllerManager.tolerations | list | `[]` |
-| imagePullPolicy | string | `"IfNotPresent"` |
-| imagePullSecrets | list | `[]` |
-| installation.namespace | string | `"function-mesh-system"` |
-| operatorImage | string | `"streamnative/function-mesh:v0.7.0"` |
-| rbac.create | bool | `true` |
+| Key | Type | Default | Description |
```

Comment:
It is recommended to delete the "Description" column.

Reply:
it's generated by helm-docs